Prelude-UIKit/lenses/UIBarButtonItemLenses.swift

```diff
@@ -7,32 +7,31 @@ public protocol UIBarButtonItemProtocol: UIBarItemProtocol {
-  func backgroundImageForState(state: UIControlState, barMetrics: UIBarMetrics) -> UIImage?
-  func setBackgroundImage(backgroundImage: UIImage?, forState state: UIControlState,
-                          style: UIBarButtonItemStyle, barMetrics: UIBarMetrics)
-  func backgroundImageForState(state: UIControlState, style: UIBarButtonItemStyle, barMetrics: UIBarMetrics)
+  func setBackgroundImage(_ backgroundImage: UIImage?, for state: UIControlState, barMetrics: UIBarMetrics)
+  func backgroundImage(for state: UIControlState, barMetrics: UIBarMetrics) -> UIImage?
+  func setBackgroundImage(_ backgroundImage: UIImage?, for state: UIControlState, style: UIBarButtonItemStyle,
+                          barMetrics: UIBarMetrics)
+  func backgroundImage(for state: UIControlState, style: UIBarButtonItemStyle, barMetrics: UIBarMetrics)
     -> UIImage?
   var tintColor: UIColor? { get set }
-  func setBackgroundVerticalPositionAdjustment(adjustment: CGFloat, forBarMetrics barMetrics: UIBarMetrics)
-  func backgroundVerticalPositionAdjustmentForBarMetrics(barMetrics: UIBarMetrics) -> CGFloat
-  func setTitlePositionAdjustment(adjustment: UIOffset, forBarMetrics barMetrics: UIBarMetrics)
-  func titlePositionAdjustmentForBarMetrics(barMetrics: UIBarMetrics) -> UIOffset
-  func setBackButtonBackgroundImage(backgroundImage: UIImage?, forState state: UIControlState,
+  func setBackgroundVerticalPositionAdjustment(_ adjustment: CGFloat, for barMetrics: UIBarMetrics)
+  func backgroundVerticalPositionAdjustment(for barMetrics: UIBarMetrics) -> CGFloat
+  func setTitlePositionAdjustment(_ adjustment: UIOffset, for barMetrics: UIBarMetrics)
+  func titlePositionAdjustment(for barMetrics: UIBarMetrics) -> UIOffset
+  func setBackButtonBackgroundImage(_ backgroundImage: UIImage?, for state: UIControlState,
                                     barMetrics: UIBarMetrics)
-  func backButtonBackgroundImageForState(state: UIControlState, barMetrics: UIBarMetrics) -> UIImage?
-  func setBackButtonTitlePositionAdjustment(adjustment: UIOffset, forBarMetrics barMetrics: UIBarMetrics)
-  func backButtonTitlePositionAdjustmentForBarMetrics(barMetrics: UIBarMetrics) -> UIOffset
-  func setBackButtonBackgroundVerticalPositionAdjustment(adjustment: CGFloat,
-                                                         forBarMetrics barMetrics: UIBarMetrics)
-  func backButtonBackgroundVerticalPositionAdjustmentForBarMetrics(barMetrics: UIBarMetrics) -> CGFloat
+  func backButtonBackgroundImage(for state: UIControlState, barMetrics: UIBarMetrics) -> UIImage?
+  func setBackButtonTitlePositionAdjustment(_ adjustment: UIOffset, for barMetrics: UIBarMetrics)
+  func backButtonTitlePositionAdjustment(for barMetrics: UIBarMetrics) -> UIOffset
+  func setBackButtonBackgroundVerticalPositionAdjustment(_ adjustment: CGFloat, for barMetrics: UIBarMetrics)
+  func backButtonBackgroundVerticalPositionAdjustment(for barMetrics: UIBarMetrics) -> CGFloat
 }
```

Comment:
im gonna save alphabetizing for another day cause it will make this diff hard to process